pkg/snapshot/store.go

```diff
@@ -929,6 +1037,17 @@ func BuildStoreFileSystem(ctx context.Context, clientset kubernetes.Interface, k
 	return &storeFileSystem, nil
 }
 
+func BuildLvpStoreFileSystem(ctx context.Context, clientset kubernetes.Interface, kotsadmNamespace string, config *types.FileSystemConfig) (*types.StoreFileSystem, error) {
+	storeFileSystem := types.StoreFileSystem{}
+
+	if config.NFS != nil && config.NFS.Path == "" {
```

Comment:
what about hostpath?

Reply:
Hmm, good point but I think we would want to require a path for hostpath, where with NFS we can probably assume we can mount to the root of the volume, right? Not sure if we have the FE validation for this though.